llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -3411,12 +3411,16 @@ unsigned AMDGPUAsmParser::getConstantBusLimit(unsigned Opcode) const {
   case AMDGPU::V_LSHLREV_B64_e64:
```

Comment:
Nit: I guess we should move this function into AMDGPUAsmUtils instead of duplicating it in AMDGPUSubtarget and AMDGPUAsmParser.

Reply:
+1 I noticed the same. It could be a separate patch.

Reply:
I will prepare new patch for this.